frontend/package.json

```diff
@@ -56,7 +56,7 @@
 		"vite-multiple-assets": "1.2.6"
 	},
 	"optionalDependencies": {
-		"wasm-pack": "0.12.1"
+		"wasm-pack": "^0.0.0"
```

Comment:
I think this was probably accidental (web package management can be quite fragile) - would it be possible to revert this change as well as the changes to the package-lock.json files?